bin/combine_spectra

```diff
@@ -106,7 +106,7 @@ class Spectrum():
             self.arffile = self._get_filename_if_not_none( infile,tab, "ancrfile", pc.read_file)
         else:
             #make sure can read it
-            tab=pc.read_file( self.arffile )
+            pc.read_file( self.arffile )
```

Comment:
look a few lines down where we check if the rmf file is readable without overwriting the variable `tab` which later is assumed to be the pha file.